Example/PaymentSheet Example/PaymentSheetUITest/PaymentSheetUITest.swift

```diff
@@ -2171,6 +2171,67 @@ class PaymentSheetDeferredServerSideUITests: PaymentSheetUITestCase {
         app.alerts.buttons["Confirm"].tap()
         XCTAssertTrue(app.staticTexts["Success!"].waitForExistence(timeout: 5.0))
     }
+
+    // MARK: - External Venmo/multiple external payment methods
+    func testExternalVenmoPaymentSheet() throws {
```

Comment:
These might be somewhat duplicative with the tests above for external PayPal, but I tried to differentiate by testing and verifying multiple external payment methods are rendered.